lnbits/core/views/generic.py

```diff
@@ -50,12 +50,9 @@ async def home(request: Request, lightning: str = ""):
 @generic_router.get("/first_install", response_class=HTMLResponse)
 async def first_install(request: Request):
     if not settings.first_install:
-        return template_renderer().TemplateResponse(
-            request,
-            "error.html",
-            {
-                "err": "Super user account has already been configured.",
-            },
+        raise HTTPException(
+            status_code=HTTPStatus.SERVICE_UNAVAILABLE,
```

Comment:
Service Unvailable is the wrong statuscode. it should only be used if the webserver is not reachable.
maybe `bad request`?